src/js/components/Layer/StyledLayer.js

```diff
@@ -50,7 +50,7 @@ export const StyledOverlay = styled.div`
   left: 0px;
   right: 0px;
   bottom: 0px;
-  background-color: ${props =>
+  background: ${props =>
     props.theme.layer.overlayBackgroundColor};
```

Comment:
I think this needs a merge.

Reply:
what do you mean ?

Reply:
Oops. I was thinking of something else. My bad.